tsconfig.json

```diff
@@ -15,7 +15,8 @@
         "esModuleInterop": true,
         "declaration": true,
         "allowSyntheticDefaultImports": true,
-        "resolveJsonModule": true
+        "resolveJsonModule": true,
+        "importsNotUsedAsValues": "error"
```

Comment:
I think we should enforce this. Thoughts again?

Reply:
Yea, I agree. We should definitely enforce this.